README.md

```diff
@@ -1,5 +1,5 @@
 [![Build Status](https://travis-ci.org/andela/project-arakari.svg?branch=develop)](https://travis-ci.org/andela/project-arakari)
-[![Coverage Status](https://coveralls.io/repos/github/andela/project-arakari/badge.svg?branch=develop)](https://coveralls.io/github/andela/project-arakari?branch=develop)
+[![Coverage Status](https://coveralls.io/repos/github/andela/project-arakari/badge.svg?branch=chore%2F137011075%2FInclude-bundling-task-in-gulp-task-runner)](https://coveralls.io/github/andela/project-arakari?branch=chore%2F137011075%2FInclude-bundling-task-in-gulp-task-runner)
```

Comment:
Since were merging to develop, I think we should leave this to point to develop.